packages/app-mobile/components/screens/ConfigScreen/FileSystemPathSelector.tsx

```diff
@@ -39,11 +39,15 @@ const useFileSystemPath = (settingId: string, updateSettingValue: UpdateSettingV
 		if (shim.mobilePlatform() === 'web') {
 			// Directory picker IDs can't include certain characters.
 			const pickerId = `setting-${settingId}`.replace(/[^a-zA-Z]/g, '_');
-			const handle = await self.showDirectoryPicker({ id: pickerId, mode: accessMode });
-			const fsDriver = shim.fsDriver() as FsDriverWeb;
-			const uri = await fsDriver.mountExternalDirectory(handle, pickerId, accessMode);
-			await updateSettingValue(settingId, uri);
-			setFileSystemPath(uri);
+			try {
+				const handle = await self.showDirectoryPicker({ id: pickerId, mode: accessMode });
+				const fsDriver = shim.fsDriver() as FsDriverWeb;
+				const uri = await fsDriver.mountExternalDirectory(handle, pickerId, accessMode);
+				await updateSettingValue(settingId, uri);
+				setFileSystemPath(uri);
+			} catch (error) {
+				return;
```

Comment:
Thank you for contributing!

It may be possible to verify that the `error` is the result of a cancellation (and `throw` it otherwise). See [the exceptions thrown by showDirectoryPicker](https://developer.mozilla.org/en-US/docs/Web/API/Window/showDirectoryPicker#exceptions). This would allow other errors (that should be reported) to be reported.

Reply:
I have read the CLA Document and I hereby sign the CLA

Reply:
recheck

Reply:
For example, checking `error.name` might be sufficient:
```suggestion
			} catch (error) {
				if (error.name === 'AbortError') {
					return;
				} else {
					throw error;
				}
```

Reply:
Thanks for the suggestion. ☺️